src/main/java/com/uber/jenkins/phabricator/conduit/DifferentialClient.java

```diff
@@ -51,33 +46,33 @@ public DifferentialClient(String diffID, LauncherFactory launcher, String condui
      * @param silent whether or not to trigger an email
      * @param action phabricator comment action, e.g. 'resign', 'reject', 'none'
      */
-    public JSONObject postComment(String revisionID, String message, boolean silent, String action) throws IOException, InterruptedException, ArcanistUsageException {
+    public JSONObject postComment(String revisionID, String message, boolean silent, String action) throws ConduitAPIException, IOException {
         Map params = new HashMap<String, String>();
         params.put("revision_id", revisionID);
         params.put("action", action);
         params.put("message", message);
-        params.put("silent", silent);
+        params.put("silent", Boolean.toString(silent));
 
         return this.callConduit("differential.createcomment", params);
     }
 
-    public JSONObject fetchDiff() throws ArcanistUsageException, IOException, InterruptedException {
+    public JSONObject fetchDiff() throws IOException, ConduitAPIException {
         Map params = new HashMap<String, String>();
-        params.put("ids", new String[]{this.diffID});
+        params.put("ids[]", diffID);
```

Comment:
yes, that is what I see as well, and what I wrote. are you getting tripped up by the brackets? or which part? the brackets is how you pass an array as a form parameter since this is no longer json encoded